spec/models/reaction_spec.rb

```diff
@@ -141,37 +141,25 @@
     let(:receiver) { build(:user) }
     let(:reaction) { build(:reaction, reactable: build(:article), user: nil) }
 
-    context "when false" do
-      it "is false when points are positive" do
-        reaction.points = 1
-        expect(reaction.skip_notification_for?(receiver)).to be(false)
-      end
-
-      it "is false when the person who reacted is not the same as the reactable owner" do
-        user_id = User.maximum(:id).to_i + 1
-        reaction.user_id = user_id
-        reaction.reactable.user_id = user_id + 1
-        expect(reaction.skip_notification_for?(user)).to be(false)
-      end
+    it "is normally false" do
+      expect(reaction.skip_notification_for?(receiver)).to be(false)
+    end
 
-      it "is false when receive_notifications is true" do
```

Comment:
this was passing because it was the default case (article.user != reaction.user, default points = 1.0 not negative) and is no longer checked in the code.